packages/flutter/lib/src/material/material_state.dart

```diff
@@ -742,7 +742,11 @@ class MaterialStatePropertyAll<T> implements MaterialStateProperty<T> {
 /// changed with [update]; it should not be modified directly.
 class MaterialStatesController extends ValueNotifier<Set<MaterialState>> {
   /// Creates a MaterialStatesController.
-  MaterialStatesController([Set<MaterialState>? value]) : super(<MaterialState>{...?value});
+  MaterialStatesController([Set<MaterialState>? value]) : super(<MaterialState>{...?value}) {
+    if (kFlutterMemoryAllocationsEnabled) {
```

Comment:
I thought super should have called this?

Reply:
hm, you are right
reverted. thank you!